AlgebraicJulia_models/3_city_seird.json

```diff
@@ -1199,7 +1199,107 @@
     }
   ],
   "ports": null,
-  "metadata": null,
+  "metadata": {
```

Comment:
I think this metadata attribute should be an array instead of an object. I caught this error after trying to parse it to test it in the HMI.